content/docs/user-guide/gems/reference/multiplayer/_index.md

```diff
@@ -14,10 +14,11 @@ For a quick introduction of the O3DE Multiplayer Gem and networking, watch the v
 
 | Topic | Description |
 |---|---|
-| [Overview](./overview) | An overview of the O3DE Multiplayer Gem, how it works, and how to use it in your project. |
-| [Project configuration](./configuration) | How to enable the O3DE Multiplayer Gem in a project. |
-| [Multiplayer Autocomponents](./autocomponents) | Automatically create components for use with the Multiplayer Gem using the AzAutoGen system. |
-| [Test Multiplayer Projects in the Editor](./test-in-editor) | Automatically launch local servers or connect to a remote persistent server when working on a multiplayer project in the O3DE Editor. |
+| [Overview](./multiplayer-gem/overview) | An overview of the O3DE Multiplayer Gem, how it works, and how to use it in your project. |
+| [Project configuration](./multiplayer-gem/configuration) | How to enable the O3DE Multiplayer Gem in a project. |
+| [Multiplayer AutoComponents](./multiplayer-gem/autocomponents) | Automatically create components for use with the Multiplayer Gem using the AzAutoGen system. |
```

Comment:
IMHO these are technical links, but they do not provide any context for why a developer wants to do this thing/

What is a "component" here? Firstly, component is standard terminology  from Lumberyard/O3DE days, but this is really an Auto component. Hard to tell from sentence the why here. Maybe these should be less technical and more informative ie "Create  components that automate network synchronization for your networking application"?

Reply:
That's really feedback for @sptramer and @willihay . I was just adding a link here though I'll soon be migrating it to AzNetworking.